spec/models/king_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe King, type: :model do
+  describe '#create' do
+    it 'creates a king in the database' do
+      king = FactoryGirl.create(:king)
+      expect(king.type == 'King').to eq(true)
+    end
+  end
+
+  describe '#valid_move?' do
+    it 'returns true if the move is diagonal up-right one square' do
+      king = FactoryGirl.create(:king, x_position: 4, y_position: 4)

```

Comment:
You could avoid specifying `x_position: 4, y_position: 4` for all of these tests by making the `x_position` and `y_position` 4 in the factory. Currently the factory is building a `king` at `x_position: 5, y_position: 1`. Since it's just for testing purposes, your factory can build the king wherever is most useful for your tests, with no impact to what happens in the actual game.